resources/sdk/purecloudjava/templates/README.mustache

```diff
@@ -170,6 +170,19 @@ UserMe me = usersApi.getUsersMe(Collections.singletonList("presence"));
 System.out.println("Hello " + me.getName());
 ```
 
+## How to Configure Retry with Java SDK
```

Comment:
These docs should be at line 130 at heading level 4 just before `#### Other ApiClient.Builder methods`, since it's documenting a feature of the ApiClient's builder. The heading should be "Setting the max retry time" to fit in with the way the other headings are worded.